modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -226,21 +227,26 @@ public function remove_core_menus() {
 		remove_submenu_page( 'options-general.php', 'sharing' );
 	}
 
-	public function add_plugin_menus() {
+	public function add_custom_menus() {
 		global $menu, $submenu;
 
-		add_menu_page( __( 'Manage Plugins', 'jetpack' ), __( 'Manage Plugins', 'jetpack' ), 'activate_plugins', 'plugins.php', '', $this->installed_plugins_icon(), 1 );
-
-		// // Count the settings page submenus, if it's zero then don't show this.
-		if ( empty( $submenu['options-general.php'] ) ) {
-			remove_menu_page( 'options-general.php' );
+		if ( $_GET[ 'post_type' ] == 'feedback' ) {
+			// there is currently no gridicon for feeback, so using dashicon
+			add_menu_page( __( 'Feedback', 'jetpack' ), __( 'Feedback', 'jetpack' ), 'edit_pages', 'edit.php?post_type=feedback', '', 'dashicons-feedback', 1 );
+			remove_submenu_page( 'edit.php?post_type=feedback', 'feedback-export');
 		} else {
-			// Rename and make sure the plugin settings menu is always last.
-			// Sneaky plugins seem to override this otherwise.
-			// Settings is always key 80.
-			$menu[80][0]                            = __( 'Plugin Settings', 'jetpack' );
-			$menu[ max( array_keys( $menu ) ) + 1 ] = $menu[80];
-			unset( $menu[80] );
+			add_menu_page( __( 'Manage Plugins', 'jetpack' ), __( 'Manage Plugins', 'jetpack' ), 'activate_plugins', 'plugins.php', '', $this->installed_plugins_icon(), 1 );
+			// Count the settings page submenus, if it's zero then don't show this.
+			if ( empty( $submenu['options-general.php'] ) ) {
+				remove_menu_page( 'options-general.php' );
```

Comment:
I see we are already removing the Settings option menu in https://github.com/Automattic/jetpack/pull/12691/files#diff-be6fcfe75adda665dbe2d6d374200406R206 so it seems this code is not causing any effect. 

Actually, after reading the code we had previously, it looks like we want to maintain the Settings option menu for Plugins (which we rename to Plugin Settings), so maybe we shouldn't always remove that option and do that only for Feedback.

Reply:
Have modified to make make removal of settings menu conditional for plugins again